tools/report-converter/codechecker_report_converter/report/__init__.py

```diff
@@ -350,23 +350,33 @@ def trim_path_prefixes(self, path_prefixes: Optional[List[str]] = None):
             event.file.trim(path_prefixes)
 
     @property
-    def files(self) -> Set[str]:
+    def files(self) -> Set[File]:
         """ Returns all referenced file paths. """
         if self.__files is not None:
             return self.__files
 
-        self.__files = {self.file.original_path}
+        self.__files = {self.file}
 
         for event in itertools.chain(
             self.bug_path_events,
             self.bug_path_positions,
             self.notes,
             self.macro_expansions
         ):
-            self.__files.add(event.file.original_path)
+            self.__files.add(event.file)
 
         return self.__files
 
+    @property
+    def trimmed_files(self) -> Set[str]:
```

Comment:
It is a little misleading, because this function doesn't return trimmed file paths necessarily, unless we call `trim()` earlier. This strange behavior also applies to `File` class too. Do we need to store trimmed file paths separately for some performance reasons? Couldn't we store the original paths and trim them in this function instead?